### PR TITLE
Feat/config tailwind

### PR DESCRIPTION
**Pull request by ViktorShevchenko**
configured tailwind.css.

### CODE REVIEW

File: components/views/Demo/Demo.tsx [hunk 1]
```diff
@@ -1,20 +1,52 @@
...
-}
+};
 
 const Demo: React.FC<Props> = ({ children }) => {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState<number>(0);
+  const [log, setLog] = useState<number[]>([]);
+
+  useEffect(() => {
+    console.log("count: ", count);
```
Comment (Security2431) [inline components/views/Demo/Demo.tsx:12]: Please, get rid of `console.log`

File: components/views/Demo/Demo.tsx [hunk 2]
```diff
@@ -1,20 +1,52 @@
...
+    };
+  }, [count]);
+
+  useEffect(() => {
+    setCount(12);
+  }, []);
+
+  useEffect(() => {
+    setLog(prevState => [...prevState, count]);
+  }, [count]);
```
Comment (Security2431) [inline components/views/Demo/Demo.tsx:25]: NIT: Testing multi lines review

File: components/views/Demo/Demo.tsx [hunk 3]
```diff
@@ -1,20 +1,52 @@
...
+import React, { useState, useEffect } from "react";
 
 type Props = {
   children: React.ReactNode;
-}
+};
 
 const Demo: React.FC<Props> = ({ children }) => {
-  const [count, setCount] = useState(0)
+  const [count, setCount] = useState<number>(0);
```
Comment (Security2431) [inline components/views/Demo/Demo.tsx:8]: Types for primitives are optional

```suggestion
  const [count, setCount] = useState(0);
```

File: package.json [hunk 4]
```diff
@@ -21,5 +22,8 @@
     "react-dom": "18.2.0",
     "tailwindcss": "3.3.2",
     "typescript": "5.0.4"
+  },
+  "devDependencies": {
+    "sass": "^1.62.1"
```
Comment (Security2431) [inline package.json]:
```suggestion
    "sass": "1.62.1"
```

File: package.json [hunk 5]
```diff
@@ -13,6 +13,7 @@
     "@types/react": "18.2.0",
     "@types/react-dom": "18.2.1",
     "autoprefixer": "10.4.14",
+    "classnames": "^2.3.2",
```
Comment (Security2431) [inline package.json:16]:
```suggestion
    "classnames": "2.3.2",
```

File: pages/index.tsx [hunk 6]
```diff
@@ -1,26 +1,18 @@
-import Demo from '@/components/views/Demo/Demo';
-import { Inter } from 'next/font/google';
-import React from 'react';
+import Icon from "@/components/shared/Icon/Icon";
+import Demo from "@/components/views/Demo/Demo";
+import { Inter } from "next/font/google";
 
-const inter = Inter({ subsets: ['latin'] });
+const inter = Inter({ subsets: ["latin"] });
```
Comment (Security2431) [inline pages/index.tsx:5]: We've already added fonts in `_app.tsx`. This line is never used
```suggestion
const inter = Inter({ subsets: ["latin"] });
```

File: pages/index.tsx [hunk 7]
```diff
@@ -1,26 +1,18 @@
-import Demo from '@/components/views/Demo/Demo';
-import { Inter } from 'next/font/google';
-import React from 'react';
+import Icon from "@/components/shared/Icon/Icon";
+import Demo from "@/components/views/Demo/Demo";
+import { Inter } from "next/font/google";
```
Comment (Security2431) [inline pages/index.tsx:3]: pls, delete one. We've already imported fonts in `_app.tsx`

File: styles/globals.scss [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+
+// body {
+//   color: rgb(var(--foreground-rgb));
+//   background: linear-gradient(
+//       to bottom,
+//       transparent,
+//       rgb(var(--background-end-rgb))
+//     )
+//     rgb(var(--background-start-rgb));
+// }
```
Comment (Security2431) [inline styles/globals.scss:31]: Please, delete ones

File: components/shared/Icon/Icon.module.scss [hunk 9]
```diff
@@ -0,0 +1,52 @@
+.icon {
+  @apply font-icomoon inline-block transition-colors;
+
+    /* use !important to prevent issues with browser extensions that change fonts */
+    speak: never;
```
Comment (Security2431) [inline components/shared/Icon/Icon.module.scss:5]: Why do we use this line here?

**Review by Security2431 — CHANGES_REQUESTED** [review]
Looks good, left some comments

File: components/views/Demo/Demo.tsx (hunk 1 above)
Comment (ViktorShevchenko) [reply components/views/Demo/Demo.tsx:12]: removed console.log

File: components/views/Demo/Demo.tsx (hunk 3 above)
Comment (ViktorShevchenko) [reply components/views/Demo/Demo.tsx:8]: removed optional <number>

File: package.json (hunk 4 above)
Comment (ViktorShevchenko) [reply package.json]: inserted the proposed "devDependencies" into package.json and removed the caret symbol before the version number

File: package.json (hunk 5 above)
Comment (ViktorShevchenko) [reply package.json:16]: inserted the "classnames" into package.json and removed the caret symbol before the version number

Comment (ViktorShevchenko) [reply package.json:16]: inserted the "classnames" into package.json and removed the caret symbol before the version number

Comment (ViktorShevchenko) [reply package.json:16]: inserted the "classnames" into package.json and removed the caret symbol before the version number

File: pages/index.tsx (hunk 6 above)
Comment (ViktorShevchenko) [reply pages/index.tsx:5]: const inter not removed. without this line there is an error in visual studio.

File: components/views/Demo/Demo.tsx (hunk 1 above)
Comment (EdPrys) [reply components/views/Demo/Demo.tsx:12]: done

File: pages/index.tsx (hunk 7 above)
Comment (ViktorShevchenko) [reply pages/index.tsx:3]: deleted "import React from 'react';"

File: styles/globals.scss (hunk 8 above)
Comment (ViktorShevchenko) [reply styles/globals.scss:31]: file 'styles/globals.scss' not found

File: components/shared/Icon/Icon.module.scss (hunk 9 above)
Comment (ViktorShevchenko) [reply components/shared/Icon/Icon.module.scss:5]: the file not found

**Pull request merged**